zigpy_cli/radio.py

```diff
@@ -60,6 +61,9 @@ async def radio(ctx, radio, port, baudrate=None, database=None):
     if baudrate is not None:
         config["device"]["baudrate"] = baudrate
 
+    if flow_control is not None:
+	config["device"]["flow_control"] = flow_control
```

Comment:
I believe this is formatted with a tab character instead of four spaces:

```suggestion
    config["device"]["flow_control"] = flow_control if flow_control != "none" else None
```

Reply:
whoops, sorry; possibly true :)